2014-03-28 14:36:16.974 [error] <0.904.0>@loadBalancerSR:terminate:206 lbsr<0.904.0>: stopping reason shutdown
2014-03-28 14:36:16.974 [error] <0.905.0> Supervisor {<0.905.0>,gateSupervisor} had child gate started with gate:start(trans, req) at undefined exit with reason {'EXIT',{badarg,[{erlang,register,[gate,<0.905.0>],[]},{gate,start,2,[{file,"src/gate.erl"},{line,18}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}} in context start_error
2014-03-28 14:36:16.975 [error] <0.898.0> Supervisor {<0.898.0>,rootSupervisor} had child gateSup started with gateSupervisor:start_link() at undefined exit with reason {shutdown,{failed_to_start_child,gate,{'EXIT',{badarg,[{erlang,register,[gate,<0.905.0>],[]},{gate,start,2,[{file,"src/gate.erl"},{line,18}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}}} in context start_error
2014-03-28 14:36:16.976 [error] <0.896.0> CRASH REPORT Process <0.896.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,gateSup,{shutdown,{failed_to_start_child,gate,{'EXIT',{badarg,[{erlang,register,[gate,<0.905.0>],[]},{gate,start,2,[{file,"src/gate.erl"},{line,18}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},...]}]}}}}}},...} in application_master:init/4 line 133
2014-03-28 14:40:13.412 [error] <0.904.0>@loadBalancerSR:terminate:206 lbsr<0.904.0>: stopping reason shutdown
2014-03-28 14:59:04.887 [error] <0.904.0>@loadBalancerSR:terminate:206 lbsr<0.904.0>: stopping reason shutdown
2014-03-28 14:59:04.888 [error] <0.905.0> Supervisor {<0.905.0>,gateSupervisor} had child gate started with gate:start(trans, req) at undefined exit with reason {'EXIT',{undef,[{gate,start,[trans,req],[]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}} in context start_error
2014-03-28 14:59:04.890 [error] <0.898.0> Supervisor {<0.898.0>,rootSupervisor} had child gateSup started with gateSupervisor:start_link() at undefined exit with reason {shutdown,{failed_to_start_child,gate,{'EXIT',{undef,[{gate,start,[trans,req],[]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}}} in context start_error
2014-03-28 14:59:04.890 [error] <0.896.0> CRASH REPORT Process <0.896.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,gateSup,{shutdown,{failed_to_start_child,gate,{'EXIT',{undef,[{gate,start,[trans,req],[]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}}}}},{bakalarka_app,start,[normal,[]]}} in application_master:init/4 line 133
2014-03-28 15:00:49.609 [error] <0.904.0>@loadBalancerSR:terminate:206 lbsr<0.904.0>: stopping reason shutdown
2014-03-28 15:00:49.612 [error] <0.905.0> Supervisor {<0.905.0>,gateSupervisor} had child gate started with gate:start([trans,req]) at undefined exit with reason {'EXIT',{badarg,[{erlang,register,[gate,<0.905.0>],[]},{gate,start,1,[{file,"src/gate.erl"},{line,24}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}} in context start_error
2014-03-28 15:00:49.612 [error] <0.898.0> Supervisor {<0.898.0>,rootSupervisor} had child gateSup started with gateSupervisor:start_link() at undefined exit with reason {shutdown,{failed_to_start_child,gate,{'EXIT',{badarg,[{erlang,register,[gate,<0.905.0>],[]},{gate,start,1,[{file,"src/gate.erl"},{line,24}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}}} in context start_error
2014-03-28 15:00:49.612 [error] <0.896.0> CRASH REPORT Process <0.896.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,gateSup,{shutdown,{failed_to_start_child,gate,{'EXIT',{badarg,[{erlang,register,[gate,<0.905.0>],[]},{gate,start,1,[{file,"src/gate.erl"},{line,24}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,310}]},{supervisor,start_children,3,[{file,"supervisor.erl"},{line,293}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,259}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},...]}]}}}}}},...} in application_master:init/4 line 133
2014-03-28 15:03:50.779 [error] <0.904.0>@loadBalancerSR:terminate:206 lbsr<0.904.0>: stopping reason shutdown
2014-03-28 16:42:51.000 [error] emulator Error in process <0.1021.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,{badmatch,{ok,[],{http_req,#Port<0.3362>,ranch_tcp,keepalive,<0.1021.0>,<<4 bytes>>,'HTTP/1.1',{{127,0,0,1},33242},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<10 bytes>>,<<99 bytes>>},{<<4 bytes>>,<<14 bytes>>},{<<6 bytes>>,<<3 bytes>>},{<<14 bytes>>,<<1 byte>>},{<<12 bytes>>,<<33 bytes>>}],[{<<14 bytes>>,0},{<<6 bytes>>,undefined},{<<14 bytes>>,0},{<<17 bytes>>,[<<8 bytes>>]},{<<12 bytes>>,{<<11 bytes>>,<<21 bytes>>,[]}},{<<17 bytes>>,undefined},{<<13 bytes>>,undefined},{<<19 bytes>>,undefined},{<<8 bytes>>,undefined... 


2014-03-28 16:42:51.002 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: {[{reason,{badmatch,{ok,[],{http_req,#Port<0.3362>,ranch_tcp,keepalive,<0.1021.0>,<<"POST">>,'HTTP/1.1',{{127,0,0,1},33242},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>},{<<"content-length">>,<<"0">>},{<<"content-type">>,<<"application/x-www-form-urlencoded">>}],[{<<"content-length">>,0},{<<"expect">>,undefined},{<<"content-length">>,0},{<<"transfer-encoding">>,[<<"identity">>]},{<<"content-type">>,{<<"application">>,<<"x-www-form-urlencoded">>,[]}},{<<"if-modified-since">>,undefined},{<<"if-none-match">>,undefined},{<<"if-unmodified-since">>,undefined},{<<"if-match">>,undefined},{<<"accept">>,[{{<<"*">>,<<"*">>,[]},1000,[]}]}],undefined,[{media_type,{<<"text">>,<<"plain">>,[]}},{charset,undefined}],done,<<>>,undefined,false,waiting,[{<<"vary">>,[<<"accept">>]},{<<"content-type">>,[<<"text">>,<<"/">>,<<"plain">>,<<>>]}],<<>>,undefined}}}},{mfa,{gate,create_paste,2}},{stacktrace,[{gate,create_paste,2,[{file,"src/gate.erl"},{line,88}]},{cowboy_rest,call,3,[{file,"src/cowboy_rest.erl"},{line,968}]},{cowboy_rest,process_content_type,3,[{file,"src/cowboy_rest.erl"},{line,773}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3362>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"POST">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},33242}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>},{<<"content-length">>,<<"0">>},{<<"content-type">>,<<"application/x-www-form-urlencoded">>}]},{p_headers,[{<<"content-type">>,{<<"application">>,<<"x-www-form-urlencoded">>,[]}},{<<"if-modified-since">>,undefined},{<<"if-none-match">>,undefined},{<<"if-unmodified-since">>,undefined},{<<"if-match">>,undefined},{<<"accept">>,[{{<<"*">>,<<"*">>,[]},1000,[]}]}]},{cookies,undefined},{meta,[{media_type,{<<"text">>,<<"plain">>,[]}},{charset,undefined}]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[{<<"vary">>,[<<"accept">>]},{<<"content-type">>,[<<"text">>,<<"/">>,<<"plain">>,<<>>]}]},{resp_body,<<>>},{onresponse,undefined}]},{state,index}],[{cowboy_rest,process_content_type,3,[{file,"src/cowboy_rest.erl"},{line,773}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-28 18:14:57.463 [error] emulator Error in process <0.1033.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,badarg},{mfa,{worker_spawner,init,3}},{stacktrace,[{erlang,link,[gate],[]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,24}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line... 


2014-03-28 18:14:57.463 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1033.0> exit with reason: {[{reason,badarg},{mfa,{worker_spawner,init,3}},{stacktrace,[{erlang,link,[gate],[]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,24}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3352>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1033.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},35045}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-28 18:51:11.884 [error] emulator Error in process <0.1014.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,undef},{mfa,{worker_spawner,init,3}},{stacktrace,[{worker_spawner,init,[{tcp,http},{http_req,#Port<0.3347>,ranch_tcp,keepalive,<0.1014.0>,<<3 bytes>>,'HTTP/1.1',{{127,0,0,1},35395},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<4 bytes>>,<<14 bytes>>},{<<10 bytes>>,<<76 bytes>>},{<<6 bytes>>,<<63 bytes>>},{<<15 bytes>>,<<14 bytes>>},{<<15 bytes>>,<<13 bytes>>},{<<10 bytes>>,<<10 bytes>>}],[{<<10 bytes>>,[<<10 bytes>>]}],undefined... 


2014-03-28 18:51:11.885 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1014.0> exit with reason: {[{reason,undef},{mfa,{worker_spawner,init,3}},{stacktrace,[{worker_spawner,init,[{tcp,http},{http_req,#Port<0.3347>,ranch_tcp,keepalive,<0.1014.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},35395},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}],[{<<"connection">>,[<<"keep-alive">>]}],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined},[]],[]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3347>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1014.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},35395}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-28 22:39:42.140 [error] emulator Error in process <0.1014.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,badarg},{mfa,{gate,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{gate,init,3,[{file,"src/gate.erl"},{line,47}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute... 


2014-03-28 22:39:42.140 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1014.0> exit with reason: {[{reason,badarg},{mfa,{gate,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{gate,init,3,[{file,"src/gate.erl"},{line,47}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3347>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1014.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},38389}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-28 22:39:42.234 [error] emulator Error in process <0.1015.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,badarg},{mfa,{gate,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{gate,init,3,[{file,"src/gate.erl"},{line,47}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute... 


2014-03-28 22:39:42.235 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1015.0> exit with reason: {[{reason,badarg},{mfa,{gate,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{gate,init,3,[{file,"src/gate.erl"},{line,47}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3348>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1015.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},38390}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/favicon.ico">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[{paste_id,<<"favicon.ico">>}]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"image/png,image/*;q=0.8,*/*;q=0.5">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-28 22:39:42.316 [error] emulator Error in process <0.1016.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,badarg},{mfa,{gate,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{gate,init,3,[{file,"src/gate.erl"},{line,47}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute... 


2014-03-28 22:39:42.317 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1016.0> exit with reason: {[{reason,badarg},{mfa,{gate,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{gate,init,3,[{file,"src/gate.erl"},{line,47}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3349>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1016.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},38391}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/favicon.ico">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[{paste_id,<<"favicon.ico">>}]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-28 23:11:57.979 [error] <0.15482.0> CRASH REPORT Process <0.15482.0> with 0 neighbours exited with reason: bad argument in call to erlang:register(ws1, <0.15482.0>) in worker_spawner:init/1 line 14 in gen_server:init_it/6 line 328
2014-03-28 23:11:57.980 [error] emulator Error in process <0.15481.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,{badmatch,{error,{badarg,[{erlang,register,[ws1,<0.15482.0>],[]},{worker_spawner,init,1,[{file,"src/worker_spawner.erl"},{line,14}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file... 


2014-03-28 23:11:57.983 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.15481.0> exit with reason: {[{reason,{badmatch,{error,{badarg,[{erlang,register,[ws1,<0.15482.0>],[]},{worker_spawner,init,1,[{file,"src/worker_spawner.erl"},{line,14}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}}},{mfa,{gate,init,3}},{stacktrace,[{gate,init,3,[{file,"src/gate.erl"},{line,49}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3354>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.15481.0>},{method,<<"POST">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},38734}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"referer">>,<<"http://localhost:8080/">>},{<<"connection">>,<<"keep-alive">>},{<<"content-type">>,<<"application/x-www-form-urlencoded">>},{<<"content-length">>,<<"7">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<"paste=v">>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-28 23:12:27.390 [error] <0.1016.0> CRASH REPORT Process <0.1016.0> with 0 neighbours exited with reason: bad argument in call to erlang:register(ws1, <0.1016.0>) in worker_spawner:init/1 line 14 in gen_server:init_it/6 line 328
2014-03-28 23:12:27.390 [error] emulator Error in process <0.1014.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,{badmatch,{error,{badarg,[{erlang,register,[ws1,<0.1016.0>],[]},{worker_spawner,init,1,[{file,"src/worker_spawner.erl"},{line,14}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file... 


2014-03-28 23:12:27.392 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1014.0> exit with reason: {[{reason,{badmatch,{error,{badarg,[{erlang,register,[ws1,<0.1016.0>],[]},{worker_spawner,init,1,[{file,"src/worker_spawner.erl"},{line,14}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}}},{mfa,{gate,init,3}},{stacktrace,[{gate,init,3,[{file,"src/gate.erl"},{line,49}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3347>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1014.0>},{method,<<"POST">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},38735}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"referer">>,<<"http://localhost:8080/">>},{<<"connection">>,<<"keep-alive">>},{<<"content-type">>,<<"application/x-www-form-urlencoded">>},{<<"content-length">>,<<"10">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<"paste=wdwd">>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-29 00:05:05.413 [error] <0.909.0> CRASH REPORT Process <0.909.0> with 0 neighbours exited with reason: no match of right hand value {error,eaddrinuse} in ranch_acceptors_sup:init/1 line 38 in gen_server:init_it/6 line 328
2014-03-29 00:05:05.414 [error] <0.904.0>@loadBalancerSR:terminate:206 lbsr<0.904.0>: stopping reason shutdown
2014-03-29 00:05:05.415 [error] <0.907.0> Supervisor {<0.907.0>,ranch_listener_sup} had child ranch_acceptors_sup started with ranch_acceptors_sup:start_link(http, 100, ranch_tcp, [{port,8080}]) at undefined exit with reason no match of right hand value {error,eaddrinuse} in ranch_acceptors_sup:init/1 line 38 in context start_error
2014-03-29 00:05:05.415 [error] <0.906.0> CRASH REPORT Process gate with 0 neighbours exited with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,start_link,[http,100,ranch_tcp,[{port,8080}],cowboy_protocol,[{env,[{dispatch,...}]}]]},...}}} in gate:init/1 line 23 in gen_server:init_it/6 line 328
2014-03-29 00:05:05.416 [error] <0.905.0> Supervisor {<0.905.0>,gateSupervisor} had child gate started with gate:start_link([trans,req]) at undefined exit with reason no match of right hand value {error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,start_link,[http,100,ranch_tcp,[{port,8080}],cowboy_protocol,[{env,[{dispatch,...}]}]]},...}}} in gate:init/1 line 23 in context start_error
2014-03-29 00:05:05.418 [error] <0.898.0> Supervisor {<0.898.0>,rootSupervisor} had child gateSup started with gateSupervisor:start_link() at undefined exit with reason {shutdown,{failed_to_start_child,gate,{{badmatch,{error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,start_link,[http,100,ranch_tcp,...]},...}}}},...}}} in context start_error
2014-03-29 00:05:05.419 [error] <0.896.0> CRASH REPORT Process <0.896.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,gateSup,{shutdown,{failed_to_start_child,gate,{{badmatch,{error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,...},...}}}},...}}}}},...} in application_master:init/4 line 133
2014-03-29 00:14:41.819 [error] <0.907.0> Supervisor {<0.907.0>,ranch_listener_sup} had child ranch_acceptors_sup started with ranch_acceptors_sup:start_link(http, 100, ranch_tcp, [{port,8080}]) at undefined exit with reason no match of right hand value {error,eaddrinuse} in ranch_acceptors_sup:init/1 line 38 in context start_error
2014-03-29 00:14:41.819 [error] <0.909.0> CRASH REPORT Process <0.909.0> with 0 neighbours exited with reason: no match of right hand value {error,eaddrinuse} in ranch_acceptors_sup:init/1 line 38 in gen_server:init_it/6 line 328
2014-03-29 00:14:41.819 [error] <0.904.0>@loadBalancerSR:terminate:206 lbsr<0.904.0>: stopping reason shutdown
2014-03-29 00:14:41.819 [error] <0.906.0> CRASH REPORT Process gate with 0 neighbours exited with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,start_link,[http,100,ranch_tcp,[{port,8080}],cowboy_protocol,[{env,[{dispatch,...}]}]]},...}}} in gate:init/1 line 23 in gen_server:init_it/6 line 328
2014-03-29 00:14:41.820 [error] <0.905.0> Supervisor {<0.905.0>,gateSupervisor} had child gate started with gate:start_link([trans,req]) at undefined exit with reason no match of right hand value {error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,start_link,[http,100,ranch_tcp,[{port,8080}],cowboy_protocol,[{env,[{dispatch,...}]}]]},...}}} in gate:init/1 line 23 in context start_error
2014-03-29 00:14:41.820 [error] <0.898.0> Supervisor {<0.898.0>,rootSupervisor} had child gateSup started with gateSupervisor:start_link() at undefined exit with reason {shutdown,{failed_to_start_child,gate,{{badmatch,{error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,start_link,[http,100,ranch_tcp,...]},...}}}},...}}} in context start_error
2014-03-29 00:14:41.821 [error] <0.896.0> CRASH REPORT Process <0.896.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,gateSup,{shutdown,{failed_to_start_child,gate,{{badmatch,{error,{{shutdown,{failed_to_start_child,ranch_acceptors_sup,{{badmatch,{error,eaddrinuse}},[{ranch_acceptors_sup,init,1,[{file,"src/ranch_acceptors_sup.erl"},{line,38}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,239}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,{ranch_listener_sup,http},{ranch_listener_sup,...},...}}}},...}}}}},...} in application_master:init/4 line 133
2014-03-29 00:15:09.593 [error] emulator Error in process <0.1014.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,undef},{mfa,{worker_spawner,init,3}},{stacktrace,[{worker_spawner,init,[{tcp,http},{http_req,#Port<0.3347>,ranch_tcp,keepalive,<0.1014.0>,<<3 bytes>>,'HTTP/1.1',{{127,0,0,1},39755},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<4 bytes>>,<<14 bytes>>},{<<10 bytes>>,<<76 bytes>>},{<<6 bytes>>,<<63 bytes>>},{<<15 bytes>>,<<14 bytes>>},{<<15 bytes>>,<<13 bytes>>},{<<10 bytes>>,<<10 bytes>>}],[{<<10 bytes>>,[<<10 bytes>>]}],undefined... 


2014-03-29 00:15:09.594 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1014.0> exit with reason: {[{reason,undef},{mfa,{worker_spawner,init,3}},{stacktrace,[{worker_spawner,init,[{tcp,http},{http_req,#Port<0.3347>,ranch_tcp,keepalive,<0.1014.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},39755},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}],[{<<"connection">>,[<<"keep-alive">>]}],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined},[]],[]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3347>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1014.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},39755}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-29 00:19:56.249 [error] emulator Error in process <0.1014.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,badarg},{mfa,{worker_spawner,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,41}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler... 


2014-03-29 00:19:56.249 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1014.0> exit with reason: {[{reason,badarg},{mfa,{worker_spawner,init,3}},{stacktrace,[{erlang,process_info,[self],[]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,41}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3347>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1014.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},39866}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>},{<<"cache-control">>,<<"max-age=0">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-03-29 00:33:14.370 [error] emulator Error in process <0.1033.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,badarg},{mfa,{worker_spawner,init,3}},{stacktrace,[{erlang,link,[gate],[]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,39}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line... 


2014-03-29 00:33:14.371 [error] <0.908.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1033.0> exit with reason: {[{reason,badarg},{mfa,{worker_spawner,init,3}},{stacktrace,[{erlang,link,[gate],[]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,39}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3352>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1033.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},39986}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 19:49:52.460 [error] emulator Error in process <0.1016.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,{badrecord,dict}},{mfa,{worker_spawner,init,3}},{stacktrace,[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{worker_helloWorld,buildResponse,2,[{file,"src/worker_helloW... 


2014-04-01 19:49:52.461 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1016.0> exit with reason: {[{reason,{badrecord,dict}},{mfa,{worker_spawner,init,3}},{stacktrace,[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{worker_helloWorld,buildResponse,2,[{file,"src/worker_helloWorld.erl"},{line,21}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1016.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43275}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 19:50:43.361 [error] <0.1040.0> CRASH REPORT Process lbss_helloWorld with 0 neighbours exited with reason: call to undefined function serviceId:init() in gen_server:init_it/6 line 328
2014-04-01 19:50:43.363 [error] <0.1038.0> Supervisor {<0.1038.0>,lbSsSupervisor} had child loadBalancerSS started with loadBalancerSS:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId,...]],...}}}) at undefined exit with reason call to undefined function serviceId:init() in context start_error
2014-04-01 19:50:43.364 [error] <0.903.0> gen_server <0.903.0> terminated with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{serviceId,init,[],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,115,115,95,104,101,108,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151
2014-04-01 19:50:43.365 [error] <0.903.0> CRASH REPORT Process sr with 0 neighbours exited with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{serviceId,init,[],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,115,115,95,104,101,108,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151 in gen_server:terminate/6 line 744
2014-04-01 19:50:43.365 [error] <0.902.0> Supervisor {<0.902.0>,serviceRegisterSupervisor} had child serviceRegister started with serviceRegister:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[[mode,...],...],...}}}) at <0.903.0> exit with reason no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{serviceId,init,[],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,115,115,95,104,101,108,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151 in context child_terminated
2014-04-01 19:55:21.623 [error] <0.1036.0> Supervisor {<0.1036.0>,lbSsSupervisor} had child loadBalancerSS started with loadBalancerSS:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId,...]],...}}}) at undefined exit with reason call to undefined function serviceServer:start_link({dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[parent,...]],...}}}) in context start_error
2014-04-01 19:55:21.623 [error] <0.1037.0> CRASH REPORT Process lbss_helloWorld with 0 neighbours exited with reason: call to undefined function serviceServer:start_link({dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[parent,...]],...}}}) in gen_server:init_it/6 line 328
2014-04-01 19:55:21.624 [error] <0.903.0> gen_server <0.903.0> terminated with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{serviceServer,start_link,[{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[parent|helloWorld]],[],[],[],[],[],[],[],[],[],[],[]}}}],[]},{helloWorld,init,0,[{file,"src/helloWorld.erl"},{line,11}]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},...}} in serviceRegister:handle_cast/2 line 151
2014-04-01 19:55:21.624 [error] <0.903.0> CRASH REPORT Process sr with 0 neighbours exited with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{serviceServer,start_link,[{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[parent|helloWorld]],[],[],[],[],[],[],[],[],[],[],[]}}}],[]},{helloWorld,init,0,[{file,"src/helloWorld.erl"},{line,11}]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},...}} in serviceRegister:handle_cast/2 line 151 in gen_server:terminate/6 line 744
2014-04-01 19:55:21.625 [error] <0.902.0> Supervisor {<0.902.0>,serviceRegisterSupervisor} had child serviceRegister started with serviceRegister:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[[mode,...],...],...}}}) at <0.903.0> exit with reason no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{serviceServer,start_link,[{dict,1,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[parent|helloWorld]],[],[],[],[],[],[],[],[],[],[],[]}}}],[]},{helloWorld,init,0,[{file,"src/helloWorld.erl"},{line,11}]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},...}} in serviceRegister:handle_cast/2 line 151 in context child_terminated
2014-04-01 19:59:35.180 [error] <0.1035.0> Supervisor {<0.1035.0>,lbSsSupervisor} had child loadBalancerSS started with loadBalancerSS:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId,...]],...}}}) at undefined exit with reason call to undefined function lager:info("helloWorld: init ~p~n", [<0.1036.0>]) in context start_error
2014-04-01 19:59:35.181 [error] <0.1036.0> CRASH REPORT Process lbss_helloWorld with 0 neighbours exited with reason: call to undefined function lager:info("helloWorld: init ~p~n", [<0.1036.0>]) in gen_server:init_it/6 line 328
2014-04-01 19:59:35.181 [error] <0.903.0> gen_server <0.903.0> terminated with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{lager,info,["helloWorld: init ~p~n",[<0.1036.0>]],[]},{helloWorld,init,0,[{file,"src/helloWorld.erl"},{line,9}]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],...},...}]},...}}} in serviceRegister:handle_cast/2 line 151
2014-04-01 19:59:35.182 [error] <0.903.0> CRASH REPORT Process sr with 0 neighbours exited with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{lager,info,["helloWorld: init ~p~n",[<0.1036.0>]],[]},{helloWorld,init,0,[{file,"src/helloWorld.erl"},{line,9}]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],...},...}]},...}}} in serviceRegister:handle_cast/2 line 151 in gen_server:terminate/6 line 744
2014-04-01 19:59:35.182 [error] <0.902.0> Supervisor {<0.902.0>,serviceRegisterSupervisor} had child serviceRegister started with serviceRegister:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[[mode,...],...],...}}}) at <0.903.0> exit with reason no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{lager,info,["helloWorld: init ~p~n",[<0.1036.0>]],[]},{helloWorld,init,0,[{file,"src/helloWorld.erl"},{line,9}]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,18}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],...},...}]},...}}} in serviceRegister:handle_cast/2 line 151 in context child_terminated
2014-04-01 20:01:38.081 [error] <0.1037.0> CRASH REPORT Process lbss_helloWorld with 1 neighbours exited with reason: call to undefined function queue:add(<0.1038.0>, {[],[]}) in gen_server:init_it/6 line 328
2014-04-01 20:01:38.083 [error] <0.1035.0> Supervisor {<0.1035.0>,lbSsSupervisor} had child loadBalancerSS started with loadBalancerSS:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId,...]],...}}}) at undefined exit with reason call to undefined function queue:add(<0.1038.0>, {[],[]}) in context start_error
2014-04-01 20:01:38.084 [error] <0.903.0> gen_server <0.903.0> terminated with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{queue,add,[<0.1038.0>,{[],[]}],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,19}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,115,115,95,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151
2014-04-01 20:01:38.084 [error] <0.903.0> CRASH REPORT Process sr with 0 neighbours exited with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{queue,add,[<0.1038.0>,{[],[]}],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,19}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,115,115,95,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151 in gen_server:terminate/6 line 744
2014-04-01 20:01:38.085 [error] <0.902.0> Supervisor {<0.902.0>,serviceRegisterSupervisor} had child serviceRegister started with serviceRegister:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[[mode,...],...],...}}}) at <0.903.0> exit with reason no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{queue,add,[<0.1038.0>,{[],[]}],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,19}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,115,115,95,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151 in context child_terminated
2014-04-01 20:03:26.418 [error] <0.1016.0> Supervisor {<0.1016.0>,lbSsSupervisor} had child loadBalancerSS started with loadBalancerSS:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId,...]],...}}}) at undefined exit with reason call to undefined function queue:add(<0.1019.0>, {[<0.1018.0>],[]}) in context start_error
2014-04-01 20:03:26.418 [error] <0.1017.0> CRASH REPORT Process lbss_helloWorld with 2 neighbours exited with reason: call to undefined function queue:add(<0.1019.0>, {[<0.1018.0>],[]}) in gen_server:init_it/6 line 328
2014-04-01 20:03:26.419 [error] <0.903.0> gen_server <0.903.0> terminated with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{queue,add,[<0.1019.0>,{[<0.1018.0>],[]}],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,21}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151
2014-04-01 20:03:26.420 [error] <0.903.0> CRASH REPORT Process sr with 0 neighbours exited with reason: no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{queue,add,[<0.1019.0>,{[<0.1018.0>],[]}],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,21}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151 in gen_server:terminate/6 line 744
2014-04-01 20:03:26.420 [error] <0.902.0> Supervisor {<0.902.0>,serviceRegisterSupervisor} had child serviceRegister started with serviceRegister:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[[mode,...],...],...}}}) at <0.903.0> exit with reason no match of right hand value {error,{{shutdown,{failed_to_start_child,loadBalancerSS,{undef,[{queue,add,[<0.1019.0>,{[<0.1018.0>],[]}],[]},{loadBalancerSS,init,1,[{file,"src/loadBalancerSS.erl"},{line,21}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,304}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]}}},{child,undefined,"lbss_helloWorld",{lbSsSupervisor,start_link,[{dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[[serviceId|helloWorld]],[[name,108,98,...]],...}}}]},...}}} in serviceRegister:handle_cast/2 line 151 in context child_terminated
2014-04-01 20:06:42.339 [error] emulator Error in process <0.1043.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,{badrecord,dict}},{mfa,{worker_spawner,init,3}},{stacktrace,[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{worker_helloWorld,buildResponse,2,[{file,"src/worker_helloW... 


2014-04-01 20:06:42.340 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1043.0> exit with reason: {[{reason,{badrecord,dict}},{mfa,{worker_spawner,init,3}},{stacktrace,[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{worker_helloWorld,buildResponse,2,[{file,"src/worker_helloWorld.erl"},{line,21}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3360>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1043.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43501}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 20:15:21.756 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: bad record dict in dict:get_slot/2 line 396
2014-04-01 20:15:21.757 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: bad record dict in dict:get_slot/2 line 396 in gen_server:terminate/6 line 744
2014-04-01 20:15:21.757 [error] <0.1022.0> gen_server <0.1022.0> terminated with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,20}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43578},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}}
2014-04-01 20:15:21.757 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,20}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43578},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}} in gen_server:terminate/6 line 744
2014-04-01 20:15:21.758 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: [{reason,{{{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,20}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43578},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]}]}},{gen_server,call,[<0.1022.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,init,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43578}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}]

2014-04-01 20:18:00.522 [error] <0.1020.0> gen_server <0.1020.0> terminated with reason: bad record dict in dict:get_slot/2 line 396
2014-04-01 20:18:00.522 [error] <0.1020.0> CRASH REPORT Process <0.1020.0> with 0 neighbours exited with reason: bad record dict in dict:get_slot/2 line 396 in gen_server:terminate/6 line 744
2014-04-01 20:18:00.523 [error] <0.1023.0> gen_server <0.1023.0> terminated with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1020.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1022.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43612},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}}
2014-04-01 20:18:00.523 [error] <0.1023.0> CRASH REPORT Process <0.1023.0> with 0 neighbours exited with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1020.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1022.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43612},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}} in gen_server:terminate/6 line 744
2014-04-01 20:18:00.524 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1022.0> exit with reason: [{reason,{{{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1020.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1022.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43612},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]}]}},{gen_server,call,[<0.1023.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,init,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1022.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43612}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}]

2014-04-01 20:20:35.185 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: bad record dict in dict:get_slot/2 line 396
2014-04-01 20:20:35.185 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: bad record dict in dict:get_slot/2 line 396 in gen_server:terminate/6 line 744
2014-04-01 20:20:35.185 [error] <0.1022.0> gen_server <0.1022.0> terminated with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43616},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}}
2014-04-01 20:20:35.185 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43616},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}} in gen_server:terminate/6 line 744
2014-04-01 20:20:35.186 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: [{reason,{{{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43616},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]}]}},{gen_server,call,[<0.1022.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,init,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43616}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}]

2014-04-01 20:22:28.943 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: bad record dict in dict:get_slot/2 line 396
2014-04-01 20:22:28.943 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: bad record dict in dict:get_slot/2 line 396 in gen_server:terminate/6 line 744
2014-04-01 20:22:28.944 [error] <0.1022.0> gen_server <0.1022.0> terminated with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43619},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}}
2014-04-01 20:22:28.944 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43619},<<"localhost">>,undefined,8080,<<"/">>,...}]}]}} in gen_server:terminate/6 line 744
2014-04-01 20:22:28.945 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: [{reason,{{{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43619},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]}]}},{gen_server,call,[<0.1022.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,init,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43619}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}]

2014-04-01 20:25:20.818 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: no function clause matching cowboy_req:chunked_response(200, [], [{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43647},...}]) line 1098
2014-04-01 20:25:20.818 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: no function clause matching cowboy_req:chunked_response(200, [], [{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43647},...}]) line 1098 in gen_server:terminate/6 line 744
2014-04-01 20:25:20.818 [error] <0.1022.0> gen_server <0.1022.0> terminated with reason: {{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43647},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/c..."},...]},...]},...}
2014-04-01 20:25:20.819 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: {{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43647},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/c..."},...]},...]},...} in gen_server:terminate/6 line 744
2014-04-01 20:25:20.820 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: [{reason,{{{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43647},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/cowboy_req.erl"},{line,1098}]},{cowboy_req,chunked_reply,3,[{file,"src/cowboy_req.erl"},{line,937}]},{helloWorld,generatePage,1,[{file,"src/helloWorld.erl"},{line,18}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,23}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43647},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]}]}},{gen_server,call,[<0.1022.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,init,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43647}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}]

2014-04-01 20:27:04.791 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: no function clause matching cowboy_req:chunked_response(200, [], [{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43652},...}]) line 1098
2014-04-01 20:27:04.791 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: no function clause matching cowboy_req:chunked_response(200, [], [{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43652},...}]) line 1098 in gen_server:terminate/6 line 744
2014-04-01 20:27:04.792 [error] <0.1022.0> gen_server <0.1022.0> terminated with reason: {{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43652},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/c..."},...]},...]},...}
2014-04-01 20:27:04.792 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: {{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43652},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/c..."},...]},...]},...} in gen_server:terminate/6 line 744
2014-04-01 20:27:04.794 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: [{reason,{{{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43652},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/cowboy_req.erl"},{line,1098}]},{cowboy_req,chunked_reply,3,[{file,"src/cowboy_req.erl"},{line,937}]},{helloWorld,generatePage,1,[{file,"src/helloWorld.erl"},{line,18}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,23}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43652},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]}]}},{gen_server,call,[<0.1022.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,init,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43652}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}]

2014-04-01 20:28:44.069 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: no function clause matching cowboy_req:chunked_response(200, [], [{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1040.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43682},...}]) line 1098
2014-04-01 20:28:44.070 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: no function clause matching cowboy_req:chunked_response(200, [], [{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1040.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43682},...}]) line 1098 in gen_server:terminate/6 line 744
2014-04-01 20:28:44.072 [error] <0.1041.0> gen_server <0.1041.0> terminated with reason: {{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1040.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43682},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/c..."},...]},...]},...}
2014-04-01 20:28:44.072 [error] <0.1041.0> CRASH REPORT Process <0.1041.0> with 0 neighbours exited with reason: {{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1040.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43682},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/c..."},...]},...]},...} in gen_server:terminate/6 line 744
2014-04-01 20:28:44.073 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1040.0> exit with reason: [{reason,{{{function_clause,[{cowboy_req,chunked_response,[200,[],[{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1040.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43682},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]],[{file,"src/cowboy_req.erl"},{line,1098}]},{cowboy_req,chunked_reply,3,[{file,"src/cowboy_req.erl"},{line,937}]},{helloWorld,generatePage,1,[{file,"src/helloWorld.erl"},{line,19}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,23}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,[{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1040.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43682},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}]}]}},{gen_server,call,[<0.1041.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,init,3}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,init,3,[{file,"src/worker_spawner.erl"},{line,14}]},{cowboy_handler,handler_init,4,[{file,"src/cowboy_handler.erl"},{line,66}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3360>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1040.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43682}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{opts,[]}]

2014-04-01 20:34:06.446 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: {[{reason,undef},{mfa,{worker_spawner,terminate,3}},{stacktrace,[{worker_spawner,terminate,[undef,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},43711},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,locked,[],<<>>,undefined},undefined],[]},{cowboy_handler,handler_terminate,4,[{file,"src/cowboy_handler.erl"},{line,291}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,121}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},43711}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined},{terminate_reason,undef}],[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,125}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 20:34:06.447 [error] emulator Error in process <0.1021.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,undef},{mfa,{worker_spawner,terminate,3}},{stacktrace,[{worker_spawner,terminate,[undef,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<3 bytes>>,'HTTP/1.1',{{127,0,0,1},43711},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<10 bytes>>,<<99 bytes>>},{<<4 bytes>>,<<14 bytes>>},{<<6 bytes>>,<<3 bytes>>}],[],undefined,[],... 


2014-04-01 21:01:58.288 [error] emulator Error in process <0.1022.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,undef},{mfa,{worker_spawner,terminate,3}},{stacktrace,[{worker_spawner,terminate,[undef,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1022.0>,<<3 bytes>>,'HTTP/1.1',{{127,0,0,1},44037},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<10 bytes>>,<<99 bytes>>},{<<4 bytes>>,<<14 bytes>>},{<<6 bytes>>,<<3 bytes>>}],[],undefined,[],... 


2014-04-01 21:01:58.289 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1022.0> exit with reason: {[{reason,undef},{mfa,{worker_spawner,terminate,3}},{stacktrace,[{worker_spawner,terminate,[undef,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1022.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},44037},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,locked,[],<<>>,undefined},undefined],[]},{cowboy_handler,handler_terminate,4,[{file,"src/cowboy_handler.erl"},{line,291}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,121}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1022.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},44037}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined},{terminate_reason,undef}],[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,125}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 21:02:32.475 [error] emulator Error in process <0.1024.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,undef},{mfa,{worker_spawner,terminate,3}},{stacktrace,[{worker_spawner,terminate,[undef,{http_req,#Port<0.3356>,ranch_tcp,keepalive,<0.1024.0>,<<3 bytes>>,'HTTP/1.1',{{127,0,0,1},44039},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<4 bytes>>,<<14 bytes>>},{<<10 bytes>>,<<76 bytes>>},{<<6 bytes>>,<<63 bytes>>},{<<15 bytes>>,<<14 bytes>>},{<<15 bytes>>,<<13 bytes>>},{<<10 bytes>>,<<10 bytes>>}],[{<<10 bytes>>,... 


2014-04-01 21:02:32.476 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1024.0> exit with reason: {[{reason,undef},{mfa,{worker_spawner,terminate,3}},{stacktrace,[{worker_spawner,terminate,[undef,{http_req,#Port<0.3356>,ranch_tcp,keepalive,<0.1024.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},44039},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}],[{<<"connection">>,[<<"keep-alive">>]}],undefined,[],waiting,<<>>,undefined,false,locked,[],<<>>,undefined},undefined],[]},{cowboy_handler,handler_terminate,4,[{file,"src/cowboy_handler.erl"},{line,291}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,121}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3356>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1024.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},44039}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"host">>,<<"localhost:8080">>},{<<"user-agent">>,<<"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:28.0) Gecko/20100101 Firefox/28.0">>},{<<"accept">>,<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{<<"accept-language">>,<<"en-US,en;q=0.5">>},{<<"accept-encoding">>,<<"gzip, deflate">>},{<<"connection">>,<<"keep-alive">>}]},{p_headers,[{<<"connection">>,[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined},{terminate_reason,undef}],[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,125}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 21:04:25.079 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1041.0> exit with reason: {[{reason,undef},{mfa,{worker_spawner,handle,2}},{stacktrace,[{worker_spawner,handle,[{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1041.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},44041},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined},undefined],[]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3360>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1041.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},44041}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}],[{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 21:04:25.080 [error] emulator Error in process <0.1041.0> on node 'bakalarka@127.0.0.1' with exit value: {[{reason,undef},{mfa,{worker_spawner,handle,2}},{stacktrace,[{worker_spawner,handle,[{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1041.0>,<<3 bytes>>,'HTTP/1.1',{{127,0,0,1},44041},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<10 bytes>>,<<99 bytes>>},{<<4 bytes>>,<<14 bytes>>},{<<6 bytes>>,<<3 bytes>>}],[],undefined,[],waiting,<<0 bytes>>,undefined... 


2014-04-01 21:07:44.250 [error] emulator Error in process <0.1041.0> on node 'bakalarka@127.0.0.1' with exit value: {{try_clause,{ok,{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1041.0>,<<3 bytes>>,'HTTP/1.1',{{127,0,0,1},44061},<<9 bytes>>,undefined,8080,<<1 byte>>,undefined,<<0 bytes>>,undefined,[],[{<<10 bytes>>,<<99 bytes>>},{<<4 bytes>>,<<14 bytes>>},{<<6 bytes>>,<<3 bytes>>}],[],undefined,[],waiting,<<0 bytes>>,undefined,false,waiting,[],<<0 bytes>>,undefined}}},[{cowboy_handler,handler_handle... 


2014-04-01 21:07:44.250 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1041.0> exit with reason: {{try_clause,{ok,{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1041.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},44061},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}},[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 21:19:02.190 [error] emulator Error in process <0.1021.0> on node 'bakalarka@127.0.0.1' with exit value: {{try_clause,{ok,undefined}},[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}


2014-04-01 21:19:02.190 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: {{try_clause,{ok,undefined}},[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 21:39:48.139 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: {{try_clause,{ok,undefined}},[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 21:39:48.139 [error] emulator Error in process <0.1021.0> on node 'bakalarka@127.0.0.1' with exit value: {{try_clause,{ok,undefined}},[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}


2014-04-01 21:43:02.961 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: {{try_clause,{ok,undefined}},[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}

2014-04-01 21:43:02.961 [error] emulator Error in process <0.1021.0> on node 'bakalarka@127.0.0.1' with exit value: {{try_clause,{ok,undefined}},[{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]}


2014-04-01 22:23:11.748 [error] <0.1346.0> CRASH REPORT Process <0.1346.0> with 0 neighbours exited with reason: bad argument in call to erlang:register(observer, <0.1346.0>) in observer_wx:init/1 line 86 in wx_object:init_it/6 line 312
2014-04-01 22:34:17.867 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: bad record dict in dict:get_slot/2 line 396
2014-04-01 22:34:17.868 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: bad record dict in dict:get_slot/2 line 396 in gen_server:terminate/6 line 744
2014-04-01 22:34:17.868 [error] <0.1022.0> gen_server <0.1022.0> terminated with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1024},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,...}}]}}
2014-04-01 22:34:17.868 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1024},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,...}}]}} in gen_server:terminate/6 line 744
2014-04-01 22:34:17.869 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: [{reason,{{{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1024},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1022.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1024}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:18.869 [error] <0.1020.0> gen_server <0.1020.0> terminated with reason: bad record dict in dict:get_slot/2 line 396
2014-04-01 22:34:18.869 [error] <0.1020.0> CRASH REPORT Process <0.1020.0> with 0 neighbours exited with reason: bad record dict in dict:get_slot/2 line 396 in gen_server:terminate/6 line 744
2014-04-01 22:34:18.869 [error] <0.1024.0> gen_server <0.1024.0> terminated with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3356>,ranch_tcp,keepalive,<0.1023.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1025},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,...}}]}}
2014-04-01 22:34:18.869 [error] <0.1024.0> CRASH REPORT Process <0.1024.0> with 0 neighbours exited with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3356>,ranch_tcp,keepalive,<0.1023.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1025},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,...}}]}} in gen_server:terminate/6 line 744
2014-04-01 22:34:18.870 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1023.0> exit with reason: [{reason,{{{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3356>,ranch_tcp,keepalive,<0.1023.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1025},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1024.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3356>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1023.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1025}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:19.868 [error] <0.1026.0> gen_server <0.1026.0> terminated with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3357>,ranch_tcp,keepalive,<0.1025.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:19.868 [error] <0.1026.0> CRASH REPORT Process <0.1026.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3357>,ranch_tcp,keepalive,<0.1025.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:19.869 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1025.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3357>,ranch_tcp,keepalive,<0.1025.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1026},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1026.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3357>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1025.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1026}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:19.869 [error] <0.1028.0> gen_server <0.1028.0> terminated with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3358>,ranch_tcp,keepalive,<0.1027.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:19.869 [error] <0.1028.0> CRASH REPORT Process <0.1028.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3358>,ranch_tcp,keepalive,<0.1027.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:19.870 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1027.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3358>,ranch_tcp,keepalive,<0.1027.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1027},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1028.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3358>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1027.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1027}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:20.868 [error] <0.1030.0> gen_server <0.1030.0> terminated with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3359>,ranch_tcp,keepalive,<0.1029.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:20.868 [error] <0.1030.0> CRASH REPORT Process <0.1030.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3359>,ranch_tcp,keepalive,<0.1029.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:20.869 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1029.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3359>,ranch_tcp,keepalive,<0.1029.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1028},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1030.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3359>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1029.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1028}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:20.870 [error] <0.1032.0> gen_server <0.1032.0> terminated with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1031.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:20.870 [error] <0.1032.0> CRASH REPORT Process <0.1032.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1031.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:20.870 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1031.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3360>,ranch_tcp,keepalive,<0.1031.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1029},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1032.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3360>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1031.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1029}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:21.868 [error] <0.1034.0> gen_server <0.1034.0> terminated with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3361>,ranch_tcp,keepalive,<0.1033.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:21.868 [error] <0.1034.0> CRASH REPORT Process <0.1034.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3361>,ranch_tcp,keepalive,<0.1033.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:21.869 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1033.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3361>,ranch_tcp,keepalive,<0.1033.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1030},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1034.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3361>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1033.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1030}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:21.870 [error] <0.1036.0> gen_server <0.1036.0> terminated with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3362>,ranch_tcp,keepalive,<0.1035.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:21.870 [error] <0.1036.0> CRASH REPORT Process <0.1036.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3362>,ranch_tcp,keepalive,<0.1035.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:21.871 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1035.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3362>,ranch_tcp,keepalive,<0.1035.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1031},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1036.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3362>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1035.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1031}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:21.871 [error] <0.1038.0> gen_server <0.1038.0> terminated with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3363>,ranch_tcp,keepalive,<0.1037.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:21.871 [error] <0.1038.0> CRASH REPORT Process <0.1038.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3363>,ranch_tcp,keepalive,<0.1037.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:21.872 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1037.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3363>,ranch_tcp,keepalive,<0.1037.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1032},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1038.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3363>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1037.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1032}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:22.868 [error] <0.1040.0> gen_server <0.1040.0> terminated with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3364>,ranch_tcp,keepalive,<0.1039.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:22.868 [error] <0.1040.0> CRASH REPORT Process <0.1040.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3364>,ranch_tcp,keepalive,<0.1039.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:22.869 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1039.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3364>,ranch_tcp,keepalive,<0.1039.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1033},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1040.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3364>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1039.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1033}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:22.870 [error] <0.1042.0> gen_server <0.1042.0> terminated with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3365>,ranch_tcp,keepalive,<0.1041.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:22.870 [error] <0.1042.0> CRASH REPORT Process <0.1042.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3365>,ranch_tcp,keepalive,<0.1041.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:22.870 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1041.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3365>,ranch_tcp,keepalive,<0.1041.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1034},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1042.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3365>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1041.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1034}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:22.872 [error] <0.1044.0> gen_server <0.1044.0> terminated with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3366>,ranch_tcp,keepalive,<0.1043.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:22.872 [error] <0.1044.0> CRASH REPORT Process <0.1044.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3366>,ranch_tcp,keepalive,<0.1043.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:22.872 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1043.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3366>,ranch_tcp,keepalive,<0.1043.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1035},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1044.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3366>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1043.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1035}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:26.317 [error] <0.1049.0> gen_server <0.1049.0> terminated with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3367>,ranch_tcp,keepalive,<0.1045.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:26.317 [error] <0.1049.0> CRASH REPORT Process <0.1049.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3367>,ranch_tcp,keepalive,<0.1045.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:26.319 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1045.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3367>,ranch_tcp,keepalive,<0.1045.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1036},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1049.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3367>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1045.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1036}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:26.320 [error] <0.1052.0> gen_server <0.1052.0> terminated with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3370>,ranch_tcp,keepalive,<0.1048.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:26.320 [error] <0.1052.0> CRASH REPORT Process <0.1052.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3370>,ranch_tcp,keepalive,<0.1048.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:26.321 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1048.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3370>,ranch_tcp,keepalive,<0.1048.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1039},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1052.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3370>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1048.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1039}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:26.322 [error] <0.1050.0> gen_server <0.1050.0> terminated with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3368>,ranch_tcp,keepalive,<0.1046.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:26.322 [error] <0.1050.0> CRASH REPORT Process <0.1050.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1019.0>, {generatePage,{http_req,#Port<0.3368>,ranch_tcp,keepalive,<0.1046.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:26.323 [error] <0.1051.0> gen_server <0.1051.0> terminated with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3369>,ranch_tcp,keepalive,<0.1047.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}})
2014-04-01 22:34:26.323 [error] <0.1051.0> CRASH REPORT Process <0.1051.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.1020.0>, {generatePage,{http_req,#Port<0.3369>,ranch_tcp,keepalive,<0.1047.0>,<<"GET">>,'HTTP/1.1',{{127,...},...},...}}) in gen_server:terminate/6 line 744
2014-04-01 22:34:26.324 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1047.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1020.0>,{generatePage,{http_req,#Port<0.3369>,ranch_tcp,keepalive,<0.1047.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1038},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1051.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3369>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1047.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1038}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:26.325 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1046.0> exit with reason: [{reason,{{noproc,{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3368>,ranch_tcp,keepalive,<0.1046.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},1037},<<"127.0.0.1">>,undefined,80,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1050.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3368>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1046.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},1037}},{host,<<"127.0.0.1">>},{host_info,undefined},{port,80},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"httperf/0.9.0">>},{<<"host">>,<<"127.0.0.1">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:34:53.328 [error] <0.903.0> gen_server <0.903.0> terminated with reason: no match of right hand value {error,{already_started,<0.1016.0>}} in serviceRegister:handle_cast/2 line 151
2014-04-01 22:34:53.328 [error] <0.903.0> CRASH REPORT Process sr with 0 neighbours exited with reason: no match of right hand value {error,{already_started,<0.1016.0>}} in serviceRegister:handle_cast/2 line 151 in gen_server:terminate/6 line 744
2014-04-01 22:34:53.328 [error] <0.902.0> Supervisor {<0.902.0>,serviceRegisterSupervisor} had child serviceRegister started with serviceRegister:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[[mode,...],...],...}}}) at <0.903.0> exit with reason no match of right hand value {error,{already_started,<0.1016.0>}} in serviceRegister:handle_cast/2 line 151 in context child_terminated
2014-04-01 22:35:12.050 [error] <0.903.0> gen_server <0.903.0> terminated with reason: bad argument in call to dict:fetch(helloWorld, {dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],...}}}) line 126 in serviceRegister:handle_call/3 line 135
2014-04-01 22:35:12.050 [error] <0.906.0>@loadBalancerSR:terminate:206 lbsr<0.906.0>: stopping reason {{badarg,[{dict,fetch,[helloWorld,{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}],[{file,"dict.erl"},{line,126}]},{serviceRegister,handle_call,3,[{file,"src/serviceRegister.erl"},{line,135}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.903.0>,{find_LbSs,helloWorld,<0.1017.0>}]}}
2014-04-01 22:35:12.050 [error] <0.903.0> CRASH REPORT Process sr with 0 neighbours exited with reason: bad argument in call to dict:fetch(helloWorld, {dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],...}}}) line 126 in serviceRegister:handle_call/3 line 135 in gen_server:terminate/6 line 744
2014-04-01 22:35:12.050 [error] <0.906.0> gen_server <0.906.0> terminated with reason: {{badarg,[{dict,fetch,[helloWorld,{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}],[{file,"dict.erl"},{line,126}]},{serviceRegister,handle_call,3,[{file,"src/serviceRegister.erl"},{line,135}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.903.0>,{find_LbSs,helloWorld,<0.1017.0>}]}}
2014-04-01 22:35:12.050 [error] <0.906.0> CRASH REPORT Process lbsr with 0 neighbours exited with reason: {{badarg,[{dict,fetch,[helloWorld,{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}],[{file,"dict.erl"},{line,126}]},{serviceRegister,handle_call,3,[{file,"src/serviceRegister.erl"},{line,135}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.903.0>,{find_LbSs,helloWorld,<0.1017.0>}]}} in gen_server:terminate/6 line 744
2014-04-01 22:35:12.051 [error] <0.902.0> Supervisor {<0.902.0>,serviceRegisterSupervisor} had child serviceRegister started with serviceRegister:start_link({dict,2,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[[mode,...],...],...}}}) at <0.903.0> exit with reason bad argument in call to dict:fetch(helloWorld, {dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],...}}}) line 126 in serviceRegister:handle_call/3 line 135 in context child_terminated
2014-04-01 22:35:12.051 [error] <0.1017.0> gen_server <0.1017.0> terminated with reason: {{{badarg,[{dict,fetch,[helloWorld,{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}],[{file,"dict.erl"},{line,126}]},{serviceRegister,handle_call,3,[{file,"src/serviceRegister.erl"},{line,135}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.903.0>,{find_LbSs,helloWorld,<0.1017.0>}]}},{gen_server,call,[lbsr,{find_LbSs,...}]}}
2014-04-01 22:35:12.052 [error] <0.905.0> Supervisor {<0.905.0>,lbsrSupervisor} had child loadBalancerSR started with loadBalancerSR:start_link() at <0.906.0> exit with reason {{badarg,[{dict,fetch,[helloWorld,{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}],[{file,"dict.erl"},{line,126}]},{serviceRegister,handle_call,3,[{file,"src/serviceRegister.erl"},{line,135}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.903.0>,{find_LbSs,helloWorld,<0.1017.0>}]}} in context child_terminated
2014-04-01 22:35:12.052 [error] <0.1017.0> CRASH REPORT Process <0.1017.0> with 0 neighbours exited with reason: {{{badarg,[{dict,fetch,[helloWorld,{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}],[{file,"dict.erl"},{line,126}]},{serviceRegister,handle_call,3,[{file,"src/serviceRegister.erl"},{line,135}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.903.0>,{find_LbSs,helloWorld,<0.1017.0>}]}},{gen_server,call,[lbsr,{find_LbSs,...}]}} in gen_server:terminate/6 line 744
2014-04-01 22:35:12.052 [error] <0.1019.0> CRASH REPORT Process lbsr with 0 neighbours exited with reason: no such process or port in call to gen_server:call(sr, {giveSRList}) in gen_server:init_it/6 line 328
2014-04-01 22:35:12.053 [error] <0.905.0> Supervisor {<0.905.0>,lbsrSupervisor} had child loadBalancerSR started with loadBalancerSR:start_link() at <0.906.0> exit with reason no such process or port in call to gen_server:call(sr, {giveSRList}) in context start_error
2014-04-01 22:35:12.054 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1016.0> exit with reason: [{reason,{{{{badarg,[{dict,fetch,[helloWorld,{dict,0,16,16,8,80,48,{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]},{{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],[]}}}],[{file,"dict.erl"},{line,126}]},{serviceRegister,handle_call,3,[{file,"src/serviceRegister.erl"},{line,135}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.903.0>,{find_LbSs,helloWorld,<0.1017.0>}]}},{gen_server,call,[lbsr,{find_LbSs,helloWorld,<0.1017.0>}]}},{gen_server,call,[<0.1017.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1016.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},45716}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

2014-04-01 22:35:12.054 [error] <0.1021.0> CRASH REPORT Process lbsr with 0 neighbours exited with reason: no such process or port in call to gen_server:call(sr, {giveSRList}) in gen_server:init_it/6 line 328
2014-04-01 22:35:12.054 [error] <0.905.0> Supervisor {<0.905.0>,lbsrSupervisor} had child loadBalancerSR started with loadBalancerSR:start_link() at {restarting,<0.906.0>} exit with reason no such process or port in call to gen_server:call(sr, {giveSRList}) in context start_error
2014-04-01 22:35:35.946 [error] <0.1019.0> gen_server <0.1019.0> terminated with reason: bad record dict in dict:get_slot/2 line 396
2014-04-01 22:35:35.946 [error] <0.1019.0> CRASH REPORT Process <0.1019.0> with 0 neighbours exited with reason: bad record dict in dict:get_slot/2 line 396 in gen_server:terminate/6 line 744
2014-04-01 22:35:35.946 [error] <0.1022.0> gen_server <0.1022.0> terminated with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},45720},<<"localhost">>,undefined,8080,<<"/">>,undefined,...}}]}}
2014-04-01 22:35:35.947 [error] <0.1022.0> CRASH REPORT Process <0.1022.0> with 0 neighbours exited with reason: {{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},45720},<<"localhost">>,undefined,8080,<<"/">>,undefined,...}}]}} in gen_server:terminate/6 line 744
2014-04-01 22:35:35.949 [error] <0.910.0> Ranch listener http had connection process started with cowboy_protocol:start_link/4 at <0.1021.0> exit with reason: [{reason,{{{{badrecord,dict},[{dict,get_slot,2,[{file,"dict.erl"},{line,396}]},{dict,fetch,2,[{file,"dict.erl"},{line,122}]},{serviceServer,handle_call,3,[{file,"src/serviceServer.erl"},{line,21}]},{gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,585}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,239}]}]},{gen_server,call,[<0.1019.0>,{generatePage,{http_req,#Port<0.3355>,ranch_tcp,keepalive,<0.1021.0>,<<"GET">>,'HTTP/1.1',{{127,0,0,1},45720},<<"localhost">>,undefined,8080,<<"/">>,undefined,<<>>,undefined,[],[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}],[],undefined,[],waiting,<<>>,undefined,false,waiting,[],<<>>,undefined}}]}},{gen_server,call,[<0.1022.0>,{generatePage,helloWorld}]}}},{mfa,{worker_spawner,handle,2}},{stacktrace,[{gen_server,call,2,[{file,"gen_server.erl"},{line,180}]},{worker_helloWorld,start_link,1,[{file,"src/worker_helloWorld.erl"},{line,13}]},{worker_spawner,handle,2,[{file,"src/worker_spawner.erl"},{line,19}]},{cowboy_handler,handler_handle,4,[{file,"src/cowboy_handler.erl"},{line,115}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,435}]}]},{req,[{socket,#Port<0.3355>},{transport,ranch_tcp},{connection,keepalive},{pid,<0.1021.0>},{method,<<"GET">>},{version,'HTTP/1.1'},{peer,{{127,0,0,1},45720}},{host,<<"localhost">>},{host_info,undefined},{port,8080},{path,<<"/">>},{path_info,undefined},{qs,<<>>},{qs_vals,undefined},{bindings,[]},{headers,[{<<"user-agent">>,<<"curl/7.22.0 (x86_64-pc-linux-gnu) libcurl/7.22.0 OpenSSL/1.0.1 zlib/1.2.3.4 libidn/1.23 librtmp/2.3">>},{<<"host">>,<<"localhost:8080">>},{<<"accept">>,<<"*/*">>}]},{p_headers,[]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{multipart,undefined},{resp_compress,false},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined}]},{state,undefined}]

